app/src/main/java/org/groundplatform/android/ui/home/mapcontainer/HomeScreenMapContainerFragment.kt

```diff
@@ -176,6 +176,8 @@ class HomeScreenMapContainerFragment : AbstractMapContainerFragment() {
     binding.bottomContainer.bringToFront()
     showDataCollectionHint()
 
+    // Always ensure that the LOIs are synced before enabling location lock. During first load of a
```

Comment:
Should the reader ensure they're synced, or is it saying the code ensures they're synced? I maybe would avoid that phrasing altogether, and just say that they're synced as a matter of fact.

Reply:
Done